src/lab_4.py

```diff
@@ -0,0 +1,74 @@
+class Shoes:
+    #створюємо клас 
+    public_number = 0 # числове поле 
+    public_string = "String" # стрічкове поле   
+    def __init__(self, name="Default Name", price=0, size=0, colour="Unknown Colour"):
+        #init - ініціалізація(налаштувати при створенні), надати початкові значення 
+        #self - перший параметр ,який надає  доступ до атрибутів 
+        #self - це параметр, що передає посилання на поточний об'єкт класу, дозволяючи взаємодіяти з його полями.
+        # приватній поля 
+        self.__name = name
+        self.__price = price
+        self.__size = size
+        self.__colour = colour 
+        # Додаткові публічні поля можуть змінюватися для кожного об'єкта
+        Shoes.public_number += 1
+        self.public_string = f"Example {Shoes.public_number}"
+    # def info(self):
+    #     return f"Shoes name: {self.__name}, Size: {self.__size}, Price: {self.__price} UAN, Colour:{self.__colour}."
+        #повернути результат виконання методу
+
+# example
+# shoes1 = shoes("Nike",5000,42,"black")
+# print(shoes1.info())
```

Comment:
remove commented code, moreover,  this comments make code non-executable

Reply:
thanks for noticing